package/src/components/ChannelList/hooks/usePaginatedChannels.ts

```diff
@@ -47,19 +47,21 @@ export const usePaginatedChannels = <
   setForceUpdate,
   sort = {},
 }: Parameters<StreamChatGenerics>) => {
-  const { client } = useChatContext<StreamChatGenerics>();
   const [channels, setChannels] = useState<Channel<StreamChatGenerics>[] | null>(null);
+  const [error, setError] = useState<Error | undefined>(undefined);
   const [staticChannelsActive, setStaticChannelsActive] = useState<boolean>(false);
+  const [activeQueryType, setActiveQueryType] = useState<QueryType | null>(null);
```

Comment:
why the change of initial state to null from "queryLocalDB"

initially we must query the local db.. crucial when there is no network and offline.. has this been tested?

Reply:
Just a un-noticed mistake, let me revert back